frontend/src/utils/functions.js

```diff
@@ -1 +1,6 @@
 export const constVal = x => () => x;
+
+export function NumberFormat(number, afterDecimal = 0, round = true) {
+    const regex = RegExp(`(^[0-9]+\.[0-9]{0,${afterDecimal}})|(^[0-9]+)`, "g")
+    return number.toFixed(afterDecimal + (round ? 0 : 1)).match(regex)[0]
```

Comment:
This is heavy handed for cases you want to round. I would test on the round param and do the regexp matching only when needed to truncate.